minai_plugin/importDataToDB.php

```diff
@@ -116,21 +116,96 @@ function importDataToDB($tableName, $folderName, $createQuery, $checkDuplicatesC
 
 }
 
+// create function to update value of updated_at column
```

Comment:
Instead of altering the existing tables, let's just drop them  (using the migrate script) and recreate them since this is not publicly released yet

Reply:
I updated create table query to add timestamp columns. Trigger and function still have to be done separately(or I don't know how to include them to create table query)